omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_9/LocationAttributeSearchHandler1_9.java

```diff
@@ -31,7 +31,7 @@
 @Component
 public class LocationAttributeSearchHandler1_9 implements SubResourceSearchHandler {
 	
-	private final static SearchConfig SEARCH_CONFIG = new SearchConfig("default", RestConstants.VERSION_1
+	private static  final   SearchConfig SEARCH_CONFIG = new SearchConfig("default", RestConstants.VERSION_1
```

Comment:
```suggestion
	private static final SearchConfig SEARCH_CONFIG = new SearchConfig("default", RestConstants.VERSION_1
```
You have too many spaces here.

Reply:
Am working out to resolve the concern